README.md

```diff
@@ -48,17 +48,28 @@ Gradle can be used to build projects developed in various programming languages.
 - Create/Clone/Download any Gradle project.
 - Edit its `build.gradle` file adding this:
 ```
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'org.sonatype.gradle.plugins:scan-gradle-plugin:0.2.0' // Update the version as needed
-    }
+plugins {
+  id 'org.sonatype.gradle.plugins.scan' version '1.0.0' // Update the version as needed
 }
+```
 
-apply plugin: 'org.sonatype.gradle.plugins.scan'
+### OSS Index
+OSS Index can be used without any extra configuration, but to avoid reaching the limit for anonymous queries every user
+is encouraged to create a free account on [OSS Index](https://ossindex.sonatype.org/user/signin) and use the credentials
+on this plugin. Cache can also be configured optionally.
+```
+ossIndexAudit {
+    username = 'email' // if not provided, an anonymous query will be made
+    password = 'pass'
```

Comment:
Suggestion: give examples that show how to get values from Environment Variables, etc... since committing these to source control would be a leaky situation for someone

Reply:
> Suggestion: give examples that show how to get values from Environment Variables, etc... since committing these to source control would be a leaky situation for someone

Done: 0d4fd8dcccff1ce2524019f88df9fcd1e6bec5f6